---
layout: archive-notitle
title: "Developments"
permalink: /development/
author_profile: true
---

## Bioinformatics Tools
### [MitoFinder](https://github.com/RemiAllio/MitoFinder){:target='_blank'}{: .discrete-link}  
_**Allio, R.**, Schomaker‐Bastos, A., Romiguier, J., Prosdocimi, F., Nabholz, B., & Delsuc, F. (2020). MitoFinder: Efficient automated large‐scale extraction of mitogenomic data in target enrichment phylogenomics. <i>Molecular Ecology Resources</i>, 20(4), 892-905. DOI: [10.1111/1755-0998.13160](https://doi.org/10.1111/1755-0998.13160){:target='_blank'}_

[Mitofinder](https://github.com/RemiAllio/MitoFinder){:target='_blank'}{: .discrete-link}   is a pipeline to **assemble** mitochondrial genomes and **annotate** mitochondrial genes from trimmed
read sequencing data.
**MitoFinder** is also designed to **find** and **annotate** mitochondrial sequences in existing genomic assemblies (generated from Hifi/PacBio/Nanopore/Illumina sequencing data...)  

<p align="center">
  <a href="https://github.com/RemiAllio/MitoFinder" target="_blank" ><img src="/images/MitoFinder.png" alt="MitoFinder_logo.png" width="500"></a>
</p>

### [PhylteR](https://github.com/damiendevienne/phylter){:target='_blank'}{: .discrete-link}
_Comte, A., Tricou, T., Tannier, E., Joseph, J., Siberchicot, A., Penel, S., **Allio, R.**, Delsuc, F., Dray, S., & de Vienne, D. M. (2023). PhylteR: efficient identification of outlier sequences in phylogenomic datasets. <i>Molecular Biology and Evolution</i>, Volume 40, Issue 11, November 2023, msad234. DOI: [10.1093/molbev/msad234](https://doi.org/10.1093/molbev/msad234){:target='_blank'}_  

[PhylteR](https://github.com/damiendevienne/phylter){:target='_blank'}{: .discrete-link}   is a tool that allows detecting, removing and visualizing outliers in phylogenomics dataset by iteratively removing taxa from gene families (gene trees) and optimizing a score of concordance between individual matrices.

<p align="center">
  <a href="https://github.com/damiendevienne/phylter" target="_blank" ><img src="/images/Phylter.png" alt="PhylteR_logo.png" width="500"></a>
</p>

## Database
### OrthoMaM
_**Allio, R.**, Delsuc, F., Belkhir, K., Douzery, E. J., Ranwez, V., & Scornavacca, C. (2023). OrthoMaM v12: a database of curated single-copy ortholog alignments and trees to study mammalian evolutionary genomics. <i>Nucleic Acids Research</i>, gkad834. DOI: [10.1093/nar/gkad834](https://doi.org/10.1093/nar/gkad834){:target='_blank'}_

[OrthoMaM](https://orthomam.mbb.cnrs.fr/){:target='_blank'}{: .discrete-link}  , a database of ORTHOlogous MAmmalian Markers describing the evolutionary dynamics of coding sequences in mammalian genomes. OrthoMaM version 12 includes 15,868 alignments of orthologous coding sequences (CDS) from the 190 complete mammalian genomes currently available.

<p align="center">
  <a href="https://orthomam.mbb.cnrs.fr/" target="_blank" ><img src="/images/omm_v11a_162tax_grey-background.png" alt="omm_v11a_162tax_ref_circular_tree_original.png" width="500"></a>
</p>


## Protocols
### ONT sequencing
_**Allio R.**, Tilak M. K., Scornavacca C., Avenant N. L., Corre E., Nabholz B., & Delsuc F. (2021). High-quality carnivore genomes from roadkill samples enable species delimitation in aardwolf and bat-eared fox. <i>eLife</i>, 10, e63167. DOI: [10.7554/eLife.63167](https://doi.org/10.7554/eLife.63167){:target='_blank'}_

[This protocol](dx.doi.org/10.17504/protocols.io.beixjcfn){:target='_blank'}{: .discrete-link}   was developed to optimize DNA sequencing from mammalian roadkill tissue samples using Oxford Nanopore Technology (ONT).  
