---
layout: archive-notitle
title: "Publications"
permalink: /publications/
author_profile: true
---
# Peer-reviewed Articles
## 2025  
13- **Allio, R.**\*, Teullet, S.\*, Lutgen, D.\*, Magdeleine, A., Koual, R., Tilak, M. K., de Thoisy, B., Emerling, C.A., Lefebure, T., & Delsuc, F. (2025). Transcriptomic data reveal divergent paths of chitinase evolution underlying dietary convergence in anteaters and pangolins. <i>Genome Biology and Evolution</i>, evaf002. DOI: [10.1093/gbe/evaf002](https://doi.org/10.1093/gbe/evaf002){:target='_blank'}  

## 2024  
12- Perrier, C., **Allio, R.**, Legeai, F., Gautier, M., Beneluz, F., Marande, W., Theron, A., Rodde, N., Herrera, M., Saune, L. and Parrinello, H. (2024). Transposable element accumulation drives genome size increase in <i>Hylesia metabus</i> (Lepidoptera: Saturniidae), an urticating moth species from South America. <i>Journal of Heredity</i>, esae069. DOI: [10.1093/jhered/esae069](https://doi.org/10.1093/jhered/esae069){:target='_blank'} 

11- **Allio, R.**, Delsuc, F., Belkhir, K., Douzery, E. J., Ranwez, V., & Scornavacca, C. (2024). OrthoMaM v12: a database of curated single-copy ortholog alignments and trees to study mammalian evolutionary genomics. <i>Nucleic Acids Research</i>, 52(D1), D529-D535. DOI: [10.1093/nar/gkad834](https://doi.org/10.1093/nar/gkad834){:target='_blank'}  

## 2023  
10- Haran, J.\*, Li, X.\*, **Allio, R.\***, Shin, S., Benoit, L., Oberprieler, R. G., Farell, B. D., Brown, S. D. J., Leschen, R. A. B., Kergoat, G. J. & McKenna, D. D. (2023). Phylogenomics illuminates the phylogeny of flower weevils (Curculioninae) and reveals ten independent origins of brood-site pollination mutualism in true weevils. <i>Proceedings of the Royal Society B</i>, 290(2008), 20230889. DOI: [10.1098/rspb.2023.0889](https://doi.org/10.1098/rspb.2023.0889){:target='_blank'}  

9- Heighton, S. P.\*, **Allio, R.\***, Murienne, J., Salmona, J., Meng, H., Scornavacca, C., Bastos, A.D.S., Njiokou, F., Pieterson, D.W., Tilak, M.-K., Luo, S.-J., Delsuc, F., & Gaubert, P. (2023). Pangolin genomes offer key insights and resources for the world's most trafficked wild mammals. <i>Molecular Biology and Evolution</i>, Volume 40, Issue 10, October 2023, msad190. DOI: [10.1093/molbev/msad190](https://doi.org/10.1093/molbev/msad190){:target='_blank'}  

8- Comte, A., Tricou, T., Tannier, E., Joseph, J., Siberchicot, A., Penel, S., **Allio, R.**, Delsuc, F., Dray, S., & de Vienne, D. M. (2023). PhylteR: efficient identification of outlier sequences in phylogenomic datasets. <i>Molecular Biology and Evolution</i>, Volume 40, Issue 11, November 2023, msad234. DOI: [10.1093/molbev/msad234](https://doi.org/10.1093/molbev/msad234){:target='_blank'}  

7- Coiro, M., **Allio, R.**, Mazet, N., Seyfullah, L. J., & Condamine, F. L. (2023). Reconciling fossils with phylogenies reveals the origin and macroevolutionary processes explaining the global cycad biodiversity. <i>New Phytologist</i>. DOI: [10.1111/nph.19010](https://doi.org/10.1111/nph.19010){:target='_blank'}  

6- Condamine, F. L., **Allio, R.**, Reboud, E. L., Dupuis, J. R., Toussaint, E. F., Mazet, N., Hu S.-J., Lewis D.S., Kunte K., Cotton A. M., & Sperling, F. A. (2023). A comprehensive phylogeny and revised taxonomy illuminate the origin and diversification of the global radiation of Papilio (Lepidoptera: Papilionidae). <i>Molecular Phylogenetics and Evolution</i>, 183, 107758. DOI: [10.1016/j.ympev.2023.107758](https://doi.org/10.1016/j.ympev.2023.107758){:target='_blank'}  

## 2021  
5- **Allio R.**, Tilak M. K., Scornavacca C., Avenant N. L., Corre E., Nabholz B., & Delsuc F. (2021). High-quality carnivore genomes from roadkill samples enable species delimitation in aardwolf and bat-eared fox. <i>eLife</i>, 10, e63167. DOI: [10.7554/eLife.63167](https://doi.org/10.7554/eLife.63167){:target='_blank'}  

## 2020  
4- **Allio R.**, Nabholz B., Wanke S., Chomicki G., Pérez-Escobar O. A., Cotton A. M., Clamens A.- L., Kergoat G. J., Sperling F. A. H. & Condamine F. L. (2020). Genome-wide macroevolutionary signatures of key innovations in butterflies colonizing new host plants. <i>Nature communications</i>, 12(1), 354. DOI: [10.1038/s41467-020-20507-3](https://doi.org/10.1038/s41467-020-20507-3){:target='_blank'}  

3- **Allio, R.**, Schomaker‐Bastos, A., Romiguier, J., Prosdocimi, F., Nabholz, B., & Delsuc, F. (2020). MitoFinder: Efficient automated large‐scale extraction of mitogenomic data in target enrichment phylogenomics. <i>Molecular Ecology Resources</i>, 20(4), 892-905. DOI: [10.1111/1755-0998.13160](https://doi.org/10.1111/1755-0998.13160){:target='_blank'}  

2- **Allio, R.**, Scornavacca, C., Nabholz, B., Clamens, A. L., Sperling, F. A., & Condamine, F. L. (2020). Whole genome shotgun phylogenomics resolves the pattern and timing of swallowtail butterfly evolution. <i>Systematic Biology</i>, 69(1), 38-60. DOI: [10.1093/sysbio/syz030](https://doi.org/10.1093/sysbio/syz030){:target='_blank'}  

## 2017  
1- **Allio, R.**, Donega, S., Galtier, N., & Nabholz, B. (2017). Large variation in the ratio of mitochondrial to nuclear mutation rate across animals: implications for genetic diversity and the use of mitochondrial DNA as a molecular marker. <i>Molecular biology and evolution</i>, 34(11), 2762-2772. DOI: [10.1093/molbev/msx197](https://doi.org/10.1093/molbev/msx197){:target='_blank'}  


# Preprints  
2- Battlay, P., Hendrickson, B. T., Mendez-Reneau, J. I., Santangelo, J. S., Albano, L. J., Wilson, J., Caizergues, A., [9 authors], **Allio, R.**, [35 authors] & Kooyers, N. J. (2024). Structural variants underlie parallel adaptation following global invasion. <i>bioRxiv</i>, 2024-07. DOI: [10.1101/2024.07.09.602765](https://doi.org/10.1101/2024.07.09.602765){:target='_blank'}  

1- Zelvelder, B., Kergoat, G. J., Benoit, L., Tsuchida, T., Haran, J., & **Allio, R.** (2024). The first example of extreme and independent host plant shifts mediated by parasitic plants in insects. <i>bioRxiv</i>, 2024-04. DOI: [10.1101/2024.04.03.587887](https://doi.org/10.1101/2024.04.03.587887){:target='_blank'}  

